src/main/java/org/broadinstitute/hellbender/tools/walkers/SplitIntervals.java

```diff
@@ -86,11 +86,14 @@
 
     public static final String MIN_CONTIG_SIZE_LONG_NAME = "min-contig-size";
 
+    public static final String INTERVAL_FILE_PREFIX_FULL_NAME = "prefix";
     public static final String INTERVAL_FILE_EXTENSION_FULL_NAME = "extension";
 
+    public static final String DEFAULT_PREFIX = "";
     public static final String PICARD_INTERVAL_FILE_EXTENSION = "interval_list";
     public static final String DEFAULT_EXTENSION = "-scattered." + PICARD_INTERVAL_FILE_EXTENSION;
 
+    public static final String INTERVAL_NUMBER_OF_DIGITS_FULL_NAME = "digits";
```

Comment:
I'd also suggest a more descriptive full name for this argument, such as `interval-file-num-digits`

Reply:
done